### PR TITLE
Added attitude and velocity std dev to `PoseMessage`.

### PR DESCRIPTION
**Pull request by adamshapiro0**
# Changes (not backwards compatible)
- Added attitude and velocity std dev
- Resized sequence number to 32b
- Renamed position std dev field and changed definition to ENU instead of ECEF
- Changed velocity definition to body frame instead of ENU

# Fixes
- Corrected missing velocity field in Python `PoseMessage` class
- Corrected `float` fields in Python messages

### CODE REVIEW

**Review by anathan — APPROVED** [review]
looks good!

File: src/point_one/fusion_engine/messages/solution.h [hunk 1]
```diff
@@ -42,28 +42,40 @@ struct PoseMessage {
   double lla_deg[3] = {NAN, NAN, NAN};
 
   /**
-   * The platform attitude (in degrees), if known, described as Euler-321 angles
-   * (yaw, pitch, roll), or `NAN` if attitude has not been initialized.
+   * The position standard deviation (in meters), resolved in the local level
```
Comment (anathan) [inline src/point_one/fusion_engine/messages/solution.h:45]: I think most people refer to this as in the Local Tangent Plane

Comment (adamshapiro0) [reply src/point_one/fusion_engine/messages/solution.h]: Yeah I've seen both but that seems more popular. Will change.

**Review by anathan — APPROVED** [review]
looks good

**Pull request merged**